packages/apps/marmalade-marketplace/src/pages/index.tsx

```diff
@@ -1,15 +1,74 @@
-import { Heading, Stack } from '@kadena/react-ui';
-import { Providers } from '@/providers/Providers';
+import * as styles from "@/styles/global.css"
 import Layout from '@/components/Layout';
+import { getSales } from "@/hooks/getSales";
+import { Button, Grid, GridItem, Link } from "@kadena/react-ui";
+import { Token } from "@/components/Token";
+import { useEffect, useState } from "react";
+import { actionBarClass, actionBarSaleActiveClass, actionBarSaleClass } from "@/styles/home.css";
+import { Providers } from "@/providers/Providers";
+
+export default function Example() {
+
+  const [saleStatus, setSaleStatus] = useState<'CREATED' | 'SOLD'>("CREATED");
+
+  const { data, loading, error, refetch } = getSales({
+    limit: 8,
+    status: saleStatus,
+    sort: [
+      {
+        field: "block",
+        direction: "desc"
+      }
+    ]
+  });
+
+  useEffect(() => {
+    refetch();
+  }, [saleStatus]);
 
-export default function Home() {
   return (
-    <Providers>
-      <Layout>
-        <Stack justifyContent={"center"} alignItems={"center"}>
-          <Heading>Marmalade Example Application</Heading>
-        </Stack>
-      </Layout>
-    </Providers>
+    <div className={styles.mainWrapperClass}>
```

Comment:
The `mainWrapperClass` here is redundant since it's already being set in the `Layout` component. This div can safely be removed